compute_endpoint/tests/unit/test_htex_regression.py

```diff
@@ -0,0 +1,95 @@
+import concurrent.futures
+import logging
+import multiprocessing
+import random
+import uuid
+
+import pytest
+from globus_compute_common import messagepack
+from globus_compute_endpoint.engines import HighThroughputEngine
+from globus_compute_sdk.serialize import ComputeSerializer
+from tests.utils import double, ez_pack_function
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def engine():
+    ep_id = uuid.uuid4()
+    engine = HighThroughputEngine(
+        label="HTEXEngine",
+        heartbeat_period=1,
+        worker_debug=True,
+    )
+    queue = multiprocessing.Queue()
+    engine.start(endpoint_id=ep_id, run_dir="/tmp", results_passthrough=queue)
+
+    yield engine
+    engine.shutdown()
+
+
+@pytest.mark.skip()
+def test_engine_submit_internal(engine):
+    "Test engine.submit with multiple engines"
+
+    param = random.randint(1, 100)
+    serializer = ComputeSerializer()
+    task_id = uuid.uuid1()
+    future = engine._submit(double, param, task_id=task_id)
+    assert isinstance(future, concurrent.futures.Future)
+
+    # The content of the future object is likely junk
+    # Do not check if the result makes sense
+    assert future.result()
+    logger.warning(f"Got result {future.result()}")
+
+    unpacked_result = messagepack.unpack(future.result())
+    assert isinstance(unpacked_result, messagepack.message_types.Result)
+    assert unpacked_result.task_id == task_id
+    logger.warning(f"Result: {unpacked_result}")
+    final_result = serializer.deserialize(unpacked_result.data)
+    assert final_result == 2 * param
+
+
+def test_engine_submit(engine):
+    q = engine.results_passthrough
+    task_id = uuid.uuid1()
+    serializer = ComputeSerializer()
+    task_body = ez_pack_function(serializer, double, (3,), {})
+    task_message = messagepack.pack(
```

Comment:
Suggested making `3` a variable, that can then be used in the test below.  Something like:
```python
...
task_arg = random.randint(1, 1000)
task_body = ez_pack_function(serializer, double, (task_num,), {})
...
assert final_result == 2 * task_arg
```
It doesn't change much semantically in the `task_body` instantiation, but makes the intent of the `assert` clearer by removing the magic number `6`.